src/ODC-By-1.0.xml

```diff
@@ -0,0 +1,504 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ODC-By-1.0"
+            name="Open Data Commons Attribution License v1.0">
```

Comment:
I think you want [`listVersionAdded="3.2"`][1] in as well.

[1]: https://github.com/spdx/license-list-XML/blob/v3.1/schema/ListedLicense.xsd#L131-L139

Reply:
done (and just realized looking back at 3.1 release licenses that I don't think we remembered to do this... )